README.md

```diff
@@ -141,15 +167,31 @@ would look like for such an application.
 Your description:
 ```
 Replace this with your answer
+A RESTful (REpresentational State Transfer) route is a conventional means for a user to interact with a Server and the related databases/objects.
 ```
 Your routes:
 ```
-The ancestors have provided an example of one route; you do the other six!
-
 GET '/warriors/:id'
-  * This is the show route, which finds a warrior by ID, and displays information about that warrior.
+  * This is the 'show' route, which finds a warrior by ID, and displays information about that warrior.
+
+GET '/warriors'
+  * This is the 'index' route, which displays a list of all warriors.
+
+GET '/warriors/new'
+  * This is the 'new' route, which creates a new warrior, via user interaction, through a form.
```

Comment:
This sounds correct.  Your use of "via user interaction" makes me feel that you have this correct.  The form presented for this routes, posts to the create route - which actually creates the warrior.